File: p001/p001_for_loop.py
# Henrik Grønbech; https://projecteuler.net/problem=1

# For loop
def sum_multiples(upper, mult0, mult1):
    total = 0
    for i in range(upper):
        if i % mult0 == 0 or i % mult1 == 0:
            total += i
    return total


if __name__ == "__main__":
    print(sum_multiples(1000, 3, 5))

File: p001/p001_for_loop.jl
# Henrik Grønbech; https://projecteuler.net/problem=1

function sum_multiples(upper, mult1, mult2)
    sum = 0
    for i = 1:upper-1
        if i % mult1 == 0 || i % mult2 == 0
            sum += i
        end
    end
    return sum
end


println(sum_multiples(1000, 3, 5))
